### PR TITLE
More clarity for quickstart.

### PR DESCRIPTION
**Pull request by anthonyshew**
The quickstart had some good opportunities to bring more clarity for first time users of tRPC. Trying to make sure first time users feel good right from the start!

## 🎯 Changes

I did do a little bit of slimy hacking here to create styling for shiki in <Tabs />.

I also added a table to give some visual acuity to the commands that were initially listed out as plain text in the original version.

## ✅ Checklist

- [ ] I have followed the steps listed in the [Contributing guide](https://github.com/trpc/trpc/blob/next/CONTRIBUTING.md).
- [ ] If necessary, I have added documentation related to the changes made.
- [ ] I have added or updated the tests related to the changes made.

### CODE REVIEW

File: www/docs/main/quickstart.mdx [hunk 1]
```diff
@@ -70,56 +71,114 @@ export type AppRouter = typeof appRouter;
...
+<Tabs>
+  <TabItem value="npm" label="npm" default>
+    <pre class="shiki nord manual-nord">
+      <div class="code-container">
+        <div class="line">
+          npm install @trpc/server@next @trpc/client@next @trpc/react-query@next
+          @trpc/next@next @tanstack/react-query
+        </div>
+      </div>
+    </pre>
```
Comment (KATT) [inline www/docs/main/quickstart.mdx:146]: This content seem to not work :(

https://www-git-shew-docsquickstart-cleanup-trpc.vercel.app/docs/v10/quickstart#installation-snippets

But I don't see why we'd need shiki in these snippets?

**Review by KATT — CHANGES_REQUESTED** [review]
Unsure why we need to add shiki into the `TabItem`s since it's all just bash stuff, can't we use ```bash`?

File: www/docs/main/quickstart.mdx (hunk 1 above)
Comment (juliusmarminge) [reply www/docs/main/quickstart.mdx:146]: What do you mean doesn't work?

CleanShot 2022-11-13 at 14 06 36

File: www/docs/main/quickstart.mdx [hunk 2]
```diff
@@ -70,56 +71,114 @@ export type AppRouter = typeof appRouter;
...
-Let's walk through the steps of building a typesafe API with tRPC. To start, this API will only contain two endpoints:
+Here are some install scripts to add tRPC to your project. These scripts include every `@trpc/*` package so feel free to remove what you don't need!
+
+<!--
+    Shiki doesn't play nice in <Tabs /> so I had to rebuild it myself.
+    The styles are all coming off of the shiki stylesheet other than
+    "manual-nord" which can be found in /src/css/custom.css.
+ -->
+
+<Tabs>
```
Comment (juliusmarminge) [inline www/docs/main/quickstart.mdx:137]: We should add these tabs to the other installation snippets too:

https://www-git-shew-docsquickstart-cleanup-trpc.vercel.app/docs/v10/nextjs#1-install-deps

https://www-git-shew-docsquickstart-cleanup-trpc.vercel.app/docs/v10/react#1-install-dependencies

https://www-git-shew-docsquickstart-cleanup-trpc.vercel.app/docs/v10/aws-lambda#1-install-deps

https://www-git-shew-docsquickstart-cleanup-trpc.vercel.app/docs/v10/express#1-install-deps

https://www-git-shew-docsquickstart-cleanup-trpc.vercel.app/docs/v10/fastify#install-dependencies

https://www-git-shew-docsquickstart-cleanup-trpc.vercel.app/docs/v10/fetch#install-dependencies

(Noticed when I grabbed these that we are not consistent with `Install deps`/`Install dependencies`)

File: www/docs/main/quickstart.mdx [hunk 3]
```diff
@@ -70,56 +71,114 @@ export type AppRouter = typeof appRouter;
...
+  <TabItem value="npm" label="npm" default>
+    <pre class="shiki nord manual-nord">
+      <div class="code-container">
+        <div class="line">
+          npm install @trpc/server@next @trpc/client@next @trpc/react-query@next
```
Comment (juliusmarminge) [inline www/docs/main/quickstart.mdx:142]: I think the rc is tagged under `@latest` so we should probably remove the `@next`.

**Review by KATT — APPROVED** [review]

File: www/docs/main/quickstart.mdx (hunk 3 above)
Comment (KATT) [reply www/docs/main/quickstart.mdx]: Thanks!

File: www/docs/main/quickstart.mdx (hunk 2 above)
Comment (KATT) [reply www/docs/main/quickstart.mdx:131]: Good shout, we should probably do an `<InstallationSnippet/>` component or something to share. I didn't do it here.

File: www/docs/main/quickstart.mdx (hunk 3 above)
Comment (KATT) [reply www/docs/main/quickstart.mdx]: Updated everywhere

**Pull request merged**